app/src/org/commcare/android/database/user/UserDatabaseUpgrader.java

```diff
@@ -214,7 +217,33 @@ private boolean upgradeSevenEight(SQLiteDatabase db, int oldVersion, int newVers
      * attributes.
      */
     private boolean upgradeEightNine(SQLiteDatabase db, int oldVersion, int newVersion) {
-        return FixtureSerializationMigration.migrateFixtureDbBytes(db, c);
+        Log.d(TAG, "starting user fixture migration");
```

Comment:
Similar to AppDatabaseUpgrader.java's update method which seems to do the ~same thing with slightly different args, it looks like?